packages/contractkit/src/wrappers/Reserve.test.ts

```diff
@@ -0,0 +1,53 @@
+import { testWithGanache } from '@celo/dev-utils/lib/ganache-test'
+import { newKitFromWeb3 } from '../kit'
+import { ReserveWrapper } from './Reserve'
+import { ReserveSpenderMultiSigWrapper } from './ReserveSpenderMultiSig'
+
+testWithGanache('Reserve Wrapper', (web3) => {
+  const kit = newKitFromWeb3(web3)
+  let accounts: string[] = []
+  let reserve: ReserveWrapper
+  let reserveSpenderMultiSig: ReserveSpenderMultiSigWrapper
+  const otherAddress = '0x91c987bf62D25945dB517BDAa840A6c661374402'
+  const otherSpender = '0x4404ac8bd8F9618D27Ad2f1485AA1B2cFD82482D'
+  // TODO @amyslawson will ganache always give same proxy address for testing?  can we ensure that?
```

Comment:
Nope, the address a contract is deployed at is a function of the deploying account's nonce and the contract code being deployed. So if the migrations are changed such that transactions are added or removed, the contract would wind up being deployed to another address.

If you need to pull the reserve spender address, you should be able to search for the `ReserveSpenderAdded` on the `Reserve`. @lucasege had to do something similar recently for testing `ReleaseGold`